src/main/java/org/opensearch/security/action/apitokens/ApiTokenAction.java

```diff
@@ -146,30 +146,32 @@ RestChannelConsumer doPrepareRequest(RestRequest request, NodeClient client) {
 
     private RestChannelConsumer handleGet(RestRequest request, NodeClient client) {
         return channel -> {
-            final XContentBuilder builder = channel.newBuilder();
-            BytesRestResponse response;
-            try {
-                Map<String, ApiToken> tokens = apiTokenRepository.getApiTokens();
-
-                builder.startArray();
-                for (ApiToken token : tokens.values()) {
-                    builder.startObject();
-                    builder.field(NAME_FIELD, token.getName());
-                    builder.field(CREATION_TIME_FIELD, token.getCreationTime().toEpochMilli());
-                    builder.field(EXPIRATION_FIELD, token.getExpiration());
-                    builder.field(CLUSTER_PERMISSIONS_FIELD, token.getClusterPermissions());
-                    builder.field(INDEX_PERMISSIONS_FIELD, token.getIndexPermissions());
-                    builder.endObject();
+            apiTokenRepository.getApiTokens(ActionListener.wrap(tokens -> {
+                try {
+                    XContentBuilder builder = channel.newBuilder();
+                    builder.startArray();
+                    for (ApiToken token : tokens.values()) {
+                        builder.startObject();
+                        builder.field(NAME_FIELD, token.getName());
+                        builder.field(CREATION_TIME_FIELD, token.getCreationTime().toEpochMilli());
```

Comment:
Can we call this issued_at to be aligned with the rfc? https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.6

Reply:
Sure Done in: [cdfc2aa](https://github.com/opensearch-project/security/pull/5147/commits/cdfc2aaf99e3aa79e6848554a4370a7eb932e317)